pkg/datapath/iptables/iptables.go

```diff
@@ -695,26 +677,99 @@ func (m *IptablesManager) installStaticProxyRules() error {
 	return err
 }
 
+var tproxyMatch = regexp.MustCompile("CILIUM_PRE_mangle .*cilium: TPROXY")
+
+// copies old proxy rules
+func (m *IptablesManager) copyProxyRules(oldChain string, match string) {
+	copyRules := func(prog string, table string, re *regexp.Regexp, match, oldChain, newChain string) {
+		output, err := runProgCombinedOutput(prog, append(m.waitArgs, "-t", table, "-S"), true)
+		if err != nil {
+			log.WithError(err).Infof("Can't list %s %s table rules", prog, table)
```

Comment:
Can we make this warning clearer to the user? There are no consequences, just a random info message that reads more like a debug message for developers.

Suggestion:

```
log.WithFields(logrus.Fields{
    "table": table,
    "prog": prog,
}).WithError(err).Warning("Cannot list rules in table. May reallocate layer 7 proxy port which could cause disruption for traffic selected by L7 policy", prog, table)
```

Then if there's a false positive in some common cases (like first startup) then maybe we can filter on the err output to skip the log in the known case like where the table is not available because we didn't create it yet?